python/ray/includes/global_state_accessor.pxi

```diff
@@ -122,6 +122,12 @@ cdef class GlobalStateAccessor:
             result = self.inner.get().AddWorkerInfo(cserialized_string)
         return result
 
+    def get_placement_group_table(self):
```

Comment:
```suggestion
    def get_all_placement_group_info(self):
```

Let's use the consistent term here.

Reply:
hi, @rkooo567 , the other naming methods in this file are all `get_object_info`<->`get_object_info`, `get_actor_table` <-> `get_actor_info`, `get_worker_table`<->`get_worker_info` ...  are we going to change them to be inconsistent (file level)